pyquil/api/_base_connection.py

```diff
@@ -284,25 +284,27 @@ def quilc_compile_payload(quil_program, isa, specs):
 
 class ForestConnection:
     @_record_call
-    def __init__(self, sync_endpoint=None, compiler_endpoint=None):
+    def __init__(self, sync_endpoint=None, compiler_endpoint=None, forest_cloud_endpoint=None):
         """
         Represents a connection to Forest containing methods to wrap all possible API endpoints.
 
         Users should not use methods from this class directly.
 
-        :param sync_endpoint: The endpoint of the server for running (small) QVM jobs
-        :param compiler_endpoint: The endpoint of the server for running (small) compiler jobs
+        :param sync_endpoint: The endpoint of the server for running QVM jobs
```

Comment:
Would it make sense to make `sync_endpoint` the forest cloud server and then add an arg `qvm_endpoint`? `sync_endpoint` to me doesn't make it clear that it's meant for a QVM, and I think the forest endpoint was passed as the `sync_endpoint` elsewhere. Would also expect the forest cloud endpoint to be the first arg

Reply:
I wanted this to be a pretty soft-touch change going into the release, so no renaming of args and no reordering. Per above and discussions with @karalekas we'll do some wider cleanup of this largely redundant class as a follow-up

Reply:
made a follow up: https://github.com/rigetticomputing/pyquil/issues/635